dipy/data/fetcher.py

```diff
@@ -591,6 +597,125 @@ def read_syn_data():
 """
 
 
+def fetch_viz_icons():
+    """ Download icons for visualization
+    """
+    url = 'https://dl.dropboxusercontent.com/u/2481924/'
```

Comment:
I have an non-free account in Dropbox and I can store big files. We also have space in Nitrc for putting up files. We should start using. Not sure the UW is the most secure.